core/service/task.go

```diff
@@ -528,9 +530,11 @@ func (t Task) StopDraggingTask(ct context.Context, taskID uint64, clientID uint6
 		entity.TaskActivity{
 			TaskID: taskID,
 			DragTaskActivity: entity.DragTaskActivity{
-				IsDragging:       false,
-				DragByUserID:     userID,
-				DraggingClientID: clientID,
+				IsDragging: false,
+				Client: entity.Client{
+					ID:     clientID,
+					UserID: userID,
+				},
```

Comment:
```suggestion
				Client: entity.Client{},
```

Not needed if task is not dragging.

Reply:
trying to apply this, but get a problem, we are setting empty client with no userid in the activity cache too, and then next time when getting activities from the gql, the resolver will still try to resolve an empty user from a client

Reply:
done